app-frontend/src/app/components/scenes/sceneItem/sceneItem.html

```diff
@@ -27,10 +27,19 @@
   </div>
   <div class="list-group-right">
     <rf-toggle-old
-      ng-if="$ctrl.onSelect && !$ctrl.isDisabled"
+      ng-if="$ctrl.onSelect && !$ctrl.isDisabled && $ctrl.hasDlPerm()"
```

Comment:
just type out `hasDownloadPermission`
Unnecessary shortenings of words are a little faster to type, but slow down reading code significantly since you have to parse what it's short for.